### PR TITLE
Ag 12640 homepage mobile and darkmode

### PR DESCRIPTION
**Pull request by markdurrant**

### CODE REVIEW

File: packages/ag-charts-website/src/pages-styles/homepage.module.scss
Comment (JamesSwinton) [inline packages/ag-charts-website/src/pages-styles/homepage.module.scss:1]: Need to add `flex-direction: column` to the maps .cardContainer selector for small screens 
```
.cardContainer {
    display: flex;
    justify-content: space-between;
    gap: $spacing-size-4;
    margin: $spacing-size-8 0;
    flex-wrap: wrap;

    @media screen and (max-width: $breakpoint-react-landing-page-medium) {
        flex-direction: column;
    }
}
```

**Comment by JamesSwinton** [comment]
Nit: Do we need to update the $breakpoint-react-landing-page-medium selector to something more specific to this page?

**Review by taktran — APPROVED** [review]
LGTM 👍🏼

File: packages/ag-charts-website/src/pages-styles/homepage.module.scss
Comment (markdurrant) [reply packages/ag-charts-website/src/pages-styles/homepage.module.scss:1]: Good shout 👍

**Comment by markdurrant** [comment]
> Nit: Do we need to update the $breakpoint-react-landing-page-medium selector to something more specific to this page?

Fair point. I think we do re-use things in a few places for breakpoints. Probably well overdue for a tidy up. 

For now I've just made them all `$breakpoint-landing-page-xxxx` so they can be used across pages. https://github.com/ag-grid/ag-charts/pull/2415/commits/cfea1e725d6500ca332def7c122198d2cbb8a86b

**Review by JamesSwinton — APPROVED** [review]
LGTM

**Pull request merged**